roles/confluent.kerberos/templates/krb5.conf.j2

```diff
@@ -1,14 +1,8 @@
-[logging]
- default = FILE:/var/log/krb5libs.log
- kdc = FILE:/var/log/krb5kdc.log
```

Comment:
I noticed that none of these files ever get written to, I think these vars are only for the kdc server, not the clients

Reply:
Correct.  I would remove them as you have.